masterlist.yaml

```diff
@@ -6572,3 +6572,24 @@ plugins:
     clean:
       - crc: 0x17ED1071
         util: '[FO4Edit v4.0.4](https://www.nexusmods.com/fallout4/mods/2737)'
+
+  - name: 'VtawWardrobe5.esp'
+    url:
+      - link: 'https://www.nexusmods.com/fallout4/mods/43844/'
+        name: 'Vtaw Wardrobe 5 - CBBE - BodySlide (Nexus Mods)'
+    msg:
+      - <<: *patch3rdParty
+        subs:
+          - '[Fusion Girl](https://www.loverslab.com/files/file/18238-zex-fusion-girl/)'
+          - '[Vtaw Wardrobe 5 - Fusion Girl Bodyslide Files](https://www.nexusmods.com/fallout4/mods/50198/)'
+        condition: 'active("FusionGirl.esp") and not checksum("Meshes/vtaw/wardrobe5/bodysuit/Gloves.nif", 54285FB0)'
```

Comment:
Since checksums are unwieldy & can easily introduce false positives, conditions like `not checksum("Meshes/vtaw/wardrobe5/bodysuit/Gloves.nif", 54285FB0)` are best avoided. Can you please convert this to check for the checksum of the default version of the file instead if possible? I.e., something like `condition: 'active("FusionGirl.esp") and checksum("Meshes/vtaw/wardrobe5/bodysuit/Gloves.nif", xxxxxxxx)'`, where `xxxxxxxx` is the checksum of the aforementioned default file.

Reply:
The condition is testing whether the mod is installed (as the patch has that checksum).  All other checksums indicate that the mod is not installed.  Are you saying that the correct determination of whether the mod should be installed is based on an arbitrary choice of a checksum from one of the possible parent checksums.  Seems a brittle way of doing things...
Also, it is not consistent with the logic used for testing an ESP.  Is the added cognitive dissonance worth it?

Reply:
Checksums unfortunately aren't as easy to deal with as just checking for if a plugin is active in the load order or for the presence of a file; they're much more volatile & require extra care to avoid false positives. It's my understanding that BodySlide edits the meshes of clothing & body mods. Given this, & given the fact that any change (excepting certain changes to the file's header & name), significant or not, will change the checksum of the file on the user's end, checking that the file isn't a specific checksum can easily lead to false positives.

If the file isn't touched by BodySlide & you think it's highly unlikely that a user would use another version of the file or have reason to edit it themselves, the condition can be kept as-is. However, if it ends up causing confusion for users for the above reasons, we'll have to change it anyway, so it'd be nice to deal with it now & save users some confusion.

> Are you saying that the correct determination of whether the mod should be installed is based on an arbitrary choice of a checksum from one of the possible parent checksums.

You can add checks for more checksums if multiple versions of the file are available like so:

```yaml
condition: 'active("FusionGirl.esp") and (checksum("Meshes/vtaw/wardrobe5/bodysuit/Gloves.nif", xxxxxxxx) or checksum("Meshes/vtaw/wardrobe5/bodysuit/Gloves.nif", yyyyyyyy) or checksum("Meshes/vtaw/wardrobe5/bodysuit/Gloves.nif", zzzzzzzz))'
```